projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/style.scss

```diff
@@ -8,13 +8,68 @@ $border_radius: math.div($bar-height, 2);
     line-height: $bar_height;
     display: flex;
     justify-content: space-between;
+    align-items: center;
     column-gap: 1em;
     margin: 8px 0 8px;
     &--letter {
+        position: relative;
+        > button {
+            all: unset;
+            font-size: var(--font-title-small);
+            font-weight: bold;
+            line-height: $bar_height;
+            color: var(--jp-green);
+            padding: 0 5px;
+            border-radius: var(--jp-border-radius);
+            cursor: pointer;
+            &:focus {
+                outline: 1px dotted var( --jp-black );
+            }
+        }
+    }
+}
+
+.boost-score-tooltip {
+    // position: absolute;
+    // top: 0;
+    // //right: 50%;
+    // right: 0;
+    // z-index: -10;
+    // width: 24px;
+    // height: 24px;
+    // pointer-events:none;
+    // @media (min-width: 783px) {
+    //     //top: 50%;
+    //     top: 0;
+    //     right: 0;
+    // }
```

Comment:
Why not remove this instead of commenting out?

Reply:
Oops! tThis I just simply overlooked. Yes, I'll remove it.